raiden/smart_contracts/ChannelManagerContract.sol

```diff
@@ -122,5 +143,16 @@ contract ChannelManagerContract {
         return data.contractExists(channel);
     }
 
+    function deleteChannel(
+        address caller_address,
+        address partner,
+        address channel_address,
+        uint caller_index,
+        uint partner_index)
+        {
```

Comment:
Indentation is wrong for this entire block. The `{` should be 4 spaces to the left and so should everything until the next `}`